content/1_General/Cpp_Command.mdx

```diff
@@ -159,29 +159,32 @@ If you want to code in (neo)vim, you can install WSL and code through WSL bash.
 
     probably suffices.
 
-4.  You should be able to compile with `g++-#`, where # is the version number (ex. 9). Running the following command
+4.  You should be able to compile with `g++-#`, where # is the version number (e.g., 10). Running the following command
 
     ```
-    g++-9 --version
+    g++-10 --version
     ```
 
     should display something like this:
 
     ```
-    g++-9 (Homebrew GCC 9.2.0_2) 9.2.0
-    Copyright (C) 2019 Free Software Foundation, Inc.
+    g++-10 (Homebrew GCC 10.2.0_2) 10.2.0
+    Copyright (C) 2020 Free Software Foundation, Inc.
     This is free software; see the source for copying conditions.  There is NO
     warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
     ```
 
-5.  If you want to be able to compile with just `g++`, create a symbolic link as mentioned [here](https://stackoverflow.com/questions/28970935/osx-replace-gcc-version-4-2-1-with-4-9-installed-via-homebrew/28982564#28982564).
+5.  If you want to be able to compile with just `g++`, write a shell alias! Put the following lines into your shell's rc file (`~/.bashrc` if you use `bash`, and `~/.zshrc` if you use `zsh`).
 
+    ```bash
+    alias g++=g++-10
     ```
-    cd /usr/local/bin
-    ln -fs g++-9 g++
-    ```
 
-    `g++ --version` should now output the same thing as `g++-9 --version`.
+    Once you do so, `g++ --version` should now output the same thing as `g++-9 --version`.
+
```

Comment:
should this be `g++-10 --version`?

Reply:
Yup, thanks for catching that! Will run through another time to make sure everything got updated.